Test/test15/ModelTest.bpl.expect

```diff
@@ -1,3 +1,6 @@
+ModelTest.bpl(9,3): Error BP5001: This assertion might not hold.
```

Comment:
I don't understand the diff in these files.  I can't see any difference.  Perhaps some hidden character ...

Reply:
The error trace appears above the model now. I think this actually makes more sense.